blocks/media/media.css

```diff
@@ -0,0 +1,230 @@
+.block.media {
+    max-width: 81.25rem;
+    width: 100%;
+    padding: 1.5rem 2.5rem;
+    margin-right: auto;
+    margin-left: auto;
+    border: var(--space-1px) solid var(--background-color-grey);
+    overflow: auto;
+}
+
+.block.media .title {
+    margin-bottom: 2.188rem;
+    position: relative;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.block.media .title::before {
+    content: "";
```

Comment:
Do we need this property ?

Reply:
<img width="1258" alt="image" src="https://github.com/aemsites/icicidirect/assets/46780200/757681d5-8d99-418d-9fcc-a154f160f17f">
yes to create strikethrough line.